tests/alive-tv/memory/alloca-address-nonnull.src.ll

```diff
@@ -1,3 +1,4 @@
+; TEST-ARGS: -smt-to=9000
```

Comment:
The updated tests usually takes less than 9 seconds (except a few), but conservatively set it to this